.github/workflows/gh-pages.yml

```diff
@@ -21,11 +21,11 @@ jobs:
       - name: Install and Build 🔧
         run: |
           npm install
-          npm run preinstall
+          npm run build:miden
```

Comment:
I don't think this will work, but we can test it by creating a dummy branch (e.g. `dominik-test-deployment`) and updating the `gh-pages.yml` file on the dummy branch to:

```
name: Deploy Miden Assembly Playground
on:
  push:
    branches:
      - dominik-test-deployment

...
```

Reply:
good point. Let's try. I do still not understand how exactly I can test this, buy now I will get into the weeds

Reply:
I am not sure how I can start the deployment tbh. I must be merged for that, right?

Reply:
ok now it works